app/src/main/java/ca/klapstein/baudit/data/Email.java

```diff
@@ -39,11 +39,21 @@ static public boolean isValid(String email) {
         return emailValidator.isValid(email);
     }
 
+    /**
+     * Get the {@code String} of the {@code Email}
+     *
+     * @return {@code String}
+     */
     @NonNull
     public String getEmail() {
         return email;
     }
 
+    /**
+     * Setter for a email
```

Comment:
I would be more descriptive here.

something like `Set the {@code String} representation of the {@code Email}.`